x/ccv/provider/keeper/migration_test.go

```diff
@@ -0,0 +1,117 @@
+package keeper_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	testutil "github.com/cosmos/interchain-security/v3/testutil/keeper"
+)
+
+func TestMigrate2To3(t *testing.T) {
+	consumerKeeper, ctx, ctrl, _ := testutil.GetProviderKeeperAndCtx(t, testutil.NewInMemKeeperParams(t))
```

Comment:
Seems that the `consumerKeeper`  and `providerKeeper` names were inverted here.

Reply:
Nice catch! https://github.com/cosmos/interchain-security/commit/9c2d1cb306998e2261c6a4a4ed21efd978dae948